src/components/markdown-renderer/replace-components/flow/flowchart/flowchart.tsx

```diff
@@ -17,32 +16,30 @@ export const FlowChart: React.FC<FlowChartProps> = ({ code }) => {
     if (diagramRef.current === null) {
       return
     }
-    const parserOutput = parse(code)
-    try {
-      parserOutput.drawSVG(diagramRef.current, {
-        'line-width': 2,
-        fill: 'none',
-        'font-size': '16px',
-        'font-family': 'Source Code Pro, twemoji, monospace'
-      })
-      setError(false)
-    } catch (error) {
-      setError(true)
-    }
-
     const currentDiagramRef = diagramRef.current
+    import('flowchart.js').then((imp) => {
+      const parserOutput = imp.parse(code)
+      try {
+        parserOutput.drawSVG(currentDiagramRef, {
+          'line-width': 2,
+          fill: 'none',
+          'font-size': '16px',
+          'font-family': 'Source Code Pro, twemoji, monospace'
+        })
+        setError(false)
+      } catch (error) {
+        setError(true)
+      }
+    }).catch(() => { console.error('error while loading flowchart.js') })
 
     return () => {
       Array.from(currentDiagramRef.children).forEach(value => value.remove())
     }
   }, [code])
 
-  if (error) {
-    return (
-      <Alert variant={'danger'}>
-        <Trans i18nKey={'renderer.flowchart.invalidSyntax'}/>
-      </Alert>
-    )
-  }
-  return <div ref={diagramRef} className={'text-center'}/>
+  return error
+    ? <Alert variant={'danger'}>
+      <Trans i18nKey={'renderer.flowchart.invalidSyntax'}/>
+    </Alert>
+    : <div ref={diagramRef} className={'text-center'}/>
```

Comment:
I'm not quite fond of changing the multi-line return statement into a variant without braces around.

Reply:
changed